deploy/helm/sumologic/values.yaml

```diff
@@ -4374,6 +4389,49 @@ otellogs:
   ##             values:
   ##             - linux
   additionalDaemonSets: {}
+  statefulset:
+    nodeSelector: {}
+    tolerations: []
+    topologySpreadConstraints: []
+    affinity: {}
+    ## Acceptable values for podAntiAffinity:
+    ## soft: specifies preferences that the scheduler will try to enforce but will not guarantee (Default)
+    ## hard: specifies rules that must be met for a pod to be scheduled onto a node
+    podAntiAffinity: "soft"
+    replicaCount: 1
+    resources:
+      limits:
+        memory: 1Gi
+        cpu: 1000m
+      requests:
+        memory: 768Mi
+        ## Warning! Increasing the CPU requests for Fluentd above 1000m might result in broken autoscaling
+        ## ref: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/best-practices.md#cpu-resources-warning
+        cpu: 500m
+    ## Option to define priorityClassName to assign a priority class to pods.
+    priorityClassName:
+
+    ## Add custom labels only to logs otel sts pods
+    podLabels: {}
+    ## Add custom annotations only to logs otel sts pods
+    podAnnotations: {}
+    ## Set securityContext for containers running in pods in otelcol-instrumentation statefulset.
+    containers:
+      otelcol:
+        securityContext: {}
+        livenessProbe:
+          initialDelaySeconds: 15
+          periodSeconds: 15
+          timeoutSeconds: 10
+          failureThreshold: 3
+        readinessProbe:
+          initialDelaySeconds: 5
+          periodSeconds: 10
+          timeoutSeconds: 3
+          failureThreshold: 3
+        startupProbe:
+          periodSeconds: 3
+          failureThreshold: 60
```

Comment:
Please doucment it in README as github actions are failing, also, I think it's not `otellogs` anymore, eventually `otellogs.cloudwatch` or `otelcloudwatch`